src/include/miopen/solver/implicitgemm_ck_util.hpp

```diff
@@ -558,9 +558,9 @@ inline size_t GetCKAlphaBetaWorkspace(const miopen::conv::ProblemDescription& pr
 inline bool CKWrwRequireWorkspace(
     size_t G, size_t C, size_t K, miopenDataType_t data_type, miopenAlphaBetaCase_t alpha_beta_case)
 {
-    auto is_odd     = [](int num) { return num % 2 != 0; };
-    int C_per_group = C / G;
-    int K_per_group = K / G;
+    auto is_odd        = [](size_t num) { return num % 2 != 0; };
```

Comment:
why are we replicating the logic. I'd like to avoid it by creating some function that is called in both places.

Reply:
Its same logic (check if odd) I was just making it more readable.

Reply:
I was hoping to place `CKWrwRequireWorkspace` in one place so that both `src/convolution_api.cpp` and `src/include/miopen/solver/implicitgemm_ck_util.hpp` could use them though.